lib/model/joinedorganization.dart

```diff
@@ -0,0 +1,146 @@
+class JoinedOrganization {
+  JoinedOrganization({
```

Comment:
@StrangeNoob If possible can you add the `required` condition for some of the parameter like id

Reply:
Sure No problem will look into more details